foundry.toml

```diff
@@ -91,6 +91,7 @@
 [rpc_endpoints]
   arbitrum = "https://arbitrum-mainnet.infura.io/v3/${API_KEY_INFURA}"
   avalanche = "https://avalanche-mainnet.infura.io/v3/${API_KEY_INFURA}"
+  blast = "${RPC_URL_BLAST}"
```

Comment:
Why not hard-code it directly here without an environment variable? I assume they have a public RPC